ivadomed/training.py

```diff
@@ -118,6 +135,11 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
                                                     num_epochs)
     logger.info("Scheduler parameters: {}".format(training_params["scheduler"]["lr_scheduler"]))
 
+    # Only call wandb methods if required params are found in the config file
+    if wandb_params is not None and wandb_params[WandbKW.WANDB_API_KEY]:
```

Comment:
Again, per @kanishk16's good suggestion, if we centralize/modularize the check into its function and encode that into the context dictionary for wandb, it might be easier than invoking this line constantly? Also... counteract the scenario of wrong/empty API key?